File: uri_exercicios_em_Cpp/beginner/selection_test1.cpp
#include <iostream>
using namespace std;

int selection_test(){
    int a,b,c,d;
    cin >> a >> b >> c >> d;

    if( b > c && d > a && c+d > a+b && c > 0 && d > 0 && a % 2 == 0){
        cout << "Valores aceitos" << endl;
        return 0;
    }

    cout << "Valores nao aceitos" << endl;
    return -1;
}

int main(){

    selection_test();

    return 0;
}

File: beecrowd_exercicios_em_python/begginner/selection_test_1.py
def test_values(x, y, w, z):
    if y > w and z > x and w + z > x + y and w > 0 and z > 0 and x % 2 == 0:
        return "Valores aceitos"

    return "Valores nao aceitos"


a, b, c, d = map(int, input().split())

print(test_values(a, b, c, d))
